src/pages/home/report/ReportActionItem.js

```diff
@@ -22,27 +14,16 @@ const propTypes = {
 };
 
 const ReportActionItem = props => (
-    <Hoverable>
-        {hovered => (
-            <View>
-                <View style={getReportActionItemContainerStyles(hovered)}>
-                    {!props.displayAsGroup
-                        ? <ReportActionItemSingle action={props.action} />
-                        : <ReportActionItemGrouped action={props.action} />}
-                </View>
-                <View style={styles.miniReportActionContextMenuWrapperStyle}>
-                    <ReportActionContextMenu
-                        reportID={props.reportID}
-                        reportActionID={props.action.sequenceNumber}
-                        isVisible={hovered}
-                        isMini
-                    />
-                </View>
-            </View>
-        )}
-    </Hoverable>
+    <>
+        {!props.displayAsGroup
+            ? <ReportActionItemSingle action={props.action} />
+            : <ReportActionItemGrouped action={props.action} />}
+    </>
 );
 
 ReportActionItem.propTypes = propTypes;
+ReportActionItem.displayName = 'ReportActionItem';
 
-export default memo(ReportActionItem, ((prevProps, nextProps) => _.isEqual(prevProps, nextProps)));
+// Must use custom isEqual callback, because the shallow comparison done by React.memo
+// would not detect changes in props.action.
+export default memo(ReportActionItem, (((prevProps, nextProps) => _.isEqual(prevProps, nextProps))));
```

Comment:
Meh, this is remnant from when I had a lot more code in this file (with some new props), but still I think it's an improvement. There's no point in having two separate comparisons like we have on master instead of just doing a deep comparison of the whole props object.

Reply:
Called this out mostly because the change doesn't seem related to the rest of your changes.

>it's an improvement. There's no point in having two separate comparisons

My intention here is that we would avoid a deep comparison if `displayAsGroup` is not equal. How does removing this improve things?

Reply:
Removed this change